sale_manual_delivery/models/sale_order_line.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models, api
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    existing_qty = fields.Float(
+        'Existing quantity',
+        compute='_compute_existing_qty',
+        help="Quantity already planned or shipped (stock movements \
+            already created)"
+    )
+
+    pending_qty_to_deliver = fields.Boolean(
+        compute='_compute_existing_qty',
+        store=True,
+        readonly=True,
+        string='There is Pending qty to add to a delivery')
+
+    @api.multi
+    def _action_procurement_create(self):
+        new_procs = self.env['procurement.order']
+        for line in self:
```

Comment:
As _action_procurement_create() is multi, it is better to use it.

```suggestion
                normal_lines = self.filtered(lambda l: l.order_id.manual_delivery)
                new_procs  = super(SaleOrderLine,
                                   normal_lines)._action_procurement_create()
```